h2o-py/h2o/h2o.py

```diff
@@ -136,7 +136,7 @@ def connection():
 def init(url=None, ip=None, port=None, name=None, https=None, cacert=None, insecure=None, username=None, password=None,
          cookies=None, proxy=None, start_h2o=True, nthreads=-1, ice_root=None, log_dir=None, log_level=None,
          max_log_file_size=None, enable_assertions=True, max_mem_size=None, min_mem_size=None, strict_version_check=None, 
-         ignore_config=False, extra_classpath=None, jvm_custom_args=None, bind_to_localhost=True, **kwargs):
+         ignore_config=False, extra_classpath=None, jvm_custom_args=None, bind_to_localhost=True, verbose = True, **kwargs):
```

Comment:
What about to rename it to show_status=True? Verbose is too general for to consider what is actually did.

Reply:
Lets keep it as it is. Since we use verbose also on internal H2OConnection